setup.py

```diff
@@ -65,7 +71,7 @@ def text(*args, **kwargs):
         f.close()
         report = validateReST(res)
         if report:
-            print report
+            logger.error(report)
```

Comment:
I don't see where you're initializing the logging system.  Without that step, all this does is print an error about no handlers being configured for logger `__main__`.

I suggest sticking with `print(report)`.

Reply:
See here: https://github.com/zopefoundation/zc.relation/pull/2/files#diff-2eeaed663bd0d25b7e608891384b7298R21

Reply:
Where are we logging this, though?
I think that all we need in setup.py is report to output in a plain print during the build of the package as it was originally.
If you're worried about the parentheses in `print(report)` you can always import print as function at the top:
```
from __future__ import print_function
```

Reply:
`logger = getLogger(__name__)` does not initialize the logging system:

```
$ python
Python 2.7.14 (default, Sep 23 2017, 22:06:14) 
[GCC 7.2.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import logging
>>> log = logging.getLogger(__name__)
>>> log.error("omg")
No handlers could be found for logger "__main__"
```